main.tf

```diff
@@ -45,7 +45,8 @@ data "aws_iam_policy_document" "policy_doc" {
 }
 
 resource "aws_cloudwatch_log_group" "log_group" {
-  name = var.name
+  name              = var.name
+  retention_in_days = var.log_retention_in_days
```

Comment:
added this line, the rest of the diff in this file is just formatting docs:https://www.terraform.io/docs/providers/aws/r/cloudwatch_log_group.html#retention_in_days